### DISCUSSION TITLE
Tuya ZG-204ZL mini PIR..

### DISCUSSION THREAD

**Discussion by jvrobert** [Koenkk/zigbee2mqtt#12033] [Devices]
Got this off Aliexpress, turns out it's not supported. I figured it would be just like the ZM-35ZH-Q but I guess not, even though it looks just like it. I tried to sort of hack it in with a custom config based on the one for that PIR but it's not working.. Anyone else seen these?

Fingerprint:
```
fingerprint: [
        {
            // The model ID from: Device with modelID 'TS0601' is not supported
            // You may need to add \u0000 at the end of the name in some cases
            modelID: 'TS0601',
            // The manufacturer name from: Device with modelID 'TS0601' is not supported.
            manufacturerName: '_TZE200_3towulqd'
        },
    ],
```

Here's the herdsman data (idle, I move, then go idle).

```
 zigbee-herdsman-converters:tuya_data_point_dump: Received DP #1 from 0xa4c1386e42614fa1 with raw data '{"dp":1,"datatype":4,"data":{"type":"Buffer","data":[0]}}': type='commandDataReport', datatype='enum', value='0', known DP# usage: ["state","moesSsystemMode","tuyaSabCOalarm","moes105DimmerState1","trsPresenceState","trsfPresenceState","haozeeSystemMode","nousTemperature","wlsWaterLeak","AM02Control","garageDoorTrigger"]
Info zigbee-herdsman-converters:tuya_data_point_dump: Received DP #12 from 0xa4c1386e42614fa1 with raw data '{"dp":12,"datatype":2,"data":{"type":"Buffer","data":[0,0,3,82]}}': type='commandDataReport', datatype='value', value='850', known DP# usage: ["haozeeChildLock","AM02Fault","garageDoorStatus"]
Info zigbee-herdsman-converters:tuya_data_point_dump: Received DP #1 from 0xa4c1386e42614fa1 with raw data '{"dp":1,"datatype":4,"data":{"type":"Buffer","data":[1]}}': type='commandDataReport', datatype='enum', value='1', known DP# usage: ["state","moesSsystemMode","tuyaSabCOalarm","moes105DimmerState1","trsPresenceState","trsfPresenceState","haozeeSystemMode","nousTemperature","wlsWaterLeak","AM02Control","garageDoorTrigger"]
```

**Comment by Impact123** [comment]
I've opened #12364 if you want to continue/test this.

**Comment by Pferdebockwurst** [comment]
> Got this off Aliexpress, turns out it's not supported. I figured it would be just like the ZM-35ZH-Q but I guess not, even though it looks just like it. I tried to sort of hack it in with a custom config based on the one for that PIR but it's not working.. Anyone else seen these?
> 
> Fingerprint:
> 
> ```
> fingerprint: [
>         {
>             // The model ID from: Device with modelID 'TS0601' is not supported
>             // You may need to add \u0000 at the end of the name in some cases
>             modelID: 'TS0601',
>             // The manufacturer name from: Device with modelID 'TS0601' is not supported.
>             manufacturerName: '_TZE200_3towulqd'
>         },
>     ],
> ```
> 
> Here's the herdsman data (idle, I move, then go idle).
> 
> ```
>  zigbee-herdsman-converters:tuya_data_point_dump: Received DP #1 from 0xa4c1386e42614fa1 with raw data '{"dp":1,"datatype":4,"data":{"type":"Buffer","data":[0]}}': type='commandDataReport', datatype='enum', value='0', known DP# usage: ["state","moesSsystemMode","tuyaSabCOalarm","moes105DimmerState1","trsPresenceState","trsfPresenceState","haozeeSystemMode","nousTemperature","wlsWaterLeak","AM02Control","garageDoorTrigger"]
> Info zigbee-herdsman-converters:tuya_data_point_dump: Received DP #12 from 0xa4c1386e42614fa1 with raw data '{"dp":12,"datatype":2,"data":{"type":"Buffer","data":[0,0,3,82]}}': type='commandDataReport', datatype='value', value='850', known DP# usage: ["haozeeChildLock","AM02Fault","garageDoorStatus"]
> Info zigbee-herdsman-converters:tuya_data_point_dump: Received DP #1 from 0xa4c1386e42614fa1 with raw data '{"dp":1,"datatype":4,"data":{"type":"Buffer","data":[1]}}': type='commandDataReport', datatype='enum', value='1', known DP# usage: ["state","moesSsystemMode","tuyaSabCOalarm","moes105DimmerState1","trsPresenceState","trsfPresenceState","haozeeSystemMode","nousTemperature","wlsWaterLeak","AM02Control","garageDoorTrigger"]
> ```

Hi, slightly off-topic but would you mind sharing a link to the offer at AliExpress? I want to buy this particular sensor, not the other Tuya ones with a similar form factor, as this one here supports a lower keep_time of 10 seconds, while the other Tuya PIRs support a lowest keep_time of 30 seconds. Unfortunately most sellers over at AliExpress don't mention the exact model number.

**Comment by Impact123** [comment]
Check my link above your post. It's where I bought mine.

**Reply by Pferdebockwurst** [reply]
Weird, the description has some pictures of the user manual, and it says that it's the model zm-35zh-q
